convert.js

```diff
@@ -204,8 +204,14 @@ var Convert = (function () {
     }
 
     function processExample(ctx, exampleNode) {
+        var exampleCode = tabsToSpaces(exampleNode.childNodes[0].nodeValue);
+        exampleCode = removeInitialLineFeed(exampleCode);
```

Comment:
@tombowers Why only initial linefeed is removed? I noticed there's also a trailing one which seems redundant.